aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -100,7 +100,52 @@ For more information, see [dotnet-counters](/dotnet/core/diagnostics/dotnet-coun
 
 ## Create custom metrics
 
-See [Create custom metrics](/dotnet/core/diagnostics/metrics#create-custom-metrics) for information on creating custom metrics.
+Metrics are created using APIs in the <xref:System.Diagnostics.Metrics> namespace. See [Create custom metrics](/dotnet/core/diagnostics/metrics-instrumentation#create-custom-metrics) for information on creating custom metrics.
+
+### Creating metrics in ASP.NET Core apps with `IMeterFactory`
+
+It's recommended to create `Meter` instances in ASP.NET Core apps with <xref:System.Diagnostics.Metrics.IMeterFactory>.
+
+ASP.NET Core registers <xref:System.Diagnostics.Metrics.IMeterFactory> in dependency injection (DI) by default. The meter factory integrates metrics with DI, making isolating and collecting metrics easy. `IMeterFactory` is especially useful for testing. It allows for multiple tests to run side-by-side and only collecting metrics values that are recorded in a test.
+
+To use `IMeterFactory` in an app, first create a type that uses `IMeterFactory` to create the app's custom metrics:
+
+```cs
+public class ContosoMetrics
+{
+    private readonly Counter<int> _productSoldCounter;
+
+    public ContosoMetrics(IMeterFactory meterFactory)
+    {
+        var meter = meterFactory.CreateMeter("Contoso.Web");
+        _productSoldCounter = meter.CreateCounter<int>("contoso.product.sold");
+    }
+
+    public void ProductSold(string productName, int quantity)
+    {
+        _productSoldCounter.Add(quantity,
+            new KeyValuePair<string, object?>("contoso.product.name", productName));
+    }
+}
+```
+
+Register the metrics type with DI in `Program.cs`:
+
+```cs
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddSingleton<ContosoMetrics>();
+```
+
+Finally, inject the metrics type and record values where needed. Because the metrics type is registered in DI it can be use with MVC controllers, minimal APIs, or any other type that is created by DI:
```

Comment:
First, `Finally` is banned. Secondly, `Finally` shouldn't be used, Finally, don't `use Finally`
```suggestion
Inject the metrics type and record values where needed. Because the metrics type is registered in DI it can be use with MVC controllers, minimal APIs, or any other type that is created by DI:
```